ts/store/reducers/backendStatus.ts

```diff
@@ -221,23 +221,6 @@ export const bancomatPayConfigSelector = createSelector(
  * based on a minumum version of the app.
  * if there is no data, false is the default value -> (LolliPOP disabled)
  */
-export const isLollipopEnabledSelector = createSelector(
-  backendStatusSelector,
-  (backendStatus): boolean =>
-    pipe(
-      backendStatus,
-      O.chainNullableK(bs => bs.config),
-      O.chainNullableK(cfg => cfg.lollipop),
-      O.chainNullableK(lp => lp.min_app_version),
```

Comment:
My two thoughts over min_app_version here (mind that they apply to the lollipop configuration **_only_**):
- if min_app_version (for lollipop) is not used anywhere else, shall we also remove it from `ts/store/reducers/__mock__/backendStatus.ts`, line 286?
- if the above holds true, should we apply the same to the io-dev-api-server file, `src/payloads/backend.ts`?  In such case, it is an activity that shall not be linked to this PR but are we already tracking such activity in our backlog?
- if the first holds true, we have the same lollipop configuration (i.e., with the `min_app_version` parameter) in the io-services-metadata repository but we must not delete it from there, otherwise previous versions of the IO application will crash. Can we track this information somewhere, somehow, so to avoid such possibility?

Reply:
@Vangaorth I'll try to give my 2 cents:
- I would not remove `min_app_version `from <ts/store/reducers/__mock__/backendStatus.ts` just to be compliant with the open api specs.
- We are not tracking the removal of the flag from the dev server, but it's a good point to do it. We'll open a JIRA task in our backlog.
- This applies to all FF in `io-services-metadata`. It's an handed down knowledge. Maybe we should write a generic topic on Confluence about it.

To me this PR is good to go the way it is.